Documentation/Packaging/MRTK_Modularization.md

```diff
@@ -0,0 +1,167 @@
+# Mixed Reality Toolkit Componentization
+
+One of the great new features of Mixed Reality Toolkit v2 is improved componentization. Wherever possible, 
+individual components are isolated from all but the core layer << >> of the foundation.
```

Comment:
><< >> [](start = 63, length = 5)

Is this intentional (i.e. I don't know if this was some formatting thing or some other key character but it looks a bit funky).

Reply:
nope. was a placeholder I forgot to remove.

thanks!